pkg/agent/controller/agent.go

```diff
@@ -631,7 +672,11 @@ func (a *Controller) getPortsForService(service *corev1.Service) []mcsv1a1.Servi
 }
 
 func (a *Controller) getObjectNameWithClusterID(name, namespace string) string {
-	return name + "-" + namespace + "-" + a.clusterID
+	return lhutil.GenerateObjectName(name, namespace, a.clusterID)
+}
+
+func GetObjectNameWithClusterID(name, namespace string, clusterID string) string {
```

Comment:
why are both a private (`getObjectNameWithClusterID` line 674) and a public (line 678) functions needed?
I was under the impression `lhutil.GenerateObjectName` replaces it so should be replaced at the call site and not via a passthrough function (i.e., caller should call `lhutil.GenerateObjectName` directly and not the local file function).